website/content/docs/release-notes/1.15.0.mdx

```diff
@@ -0,0 +1,241 @@
+
+## Known issues and breaking changes
+
+None.
+
+## Vault companion updates
+
+Companion updates are Vault updates that live outside the main Vault binary.
+
+<table>
+  <thead>
+    <tr>
+      <th style={{verticalAlign: 'middle'}}>Release</th>
+      <th style={{verticalAlign: 'middle'}}>Update</th>
+      <th style={{verticalAlign: 'middle'}}>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      Vault Secrets Operator
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>GA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Run the Vault Secrets Operator (v0.2.0) on Red Hat OpenShift.
+      <br /><br />
+      Learn more: <a href="/vault/docs/platform/k8s/vso/openshift">Vault Secrets Operator</a>
+    </td>
+  </tr>
+  </tbody>
+</table>
+
+## Core updates
+
+Follow the learn more links for more information, or browse the list of
+[Vault tutorials updated to highlight changes for the most recent GA release](/vault/tutorials/new-release).
+
+<table>
+  <thead>
+    <tr>
+      <th style={{verticalAlign: 'middle'}}>Release</th>
+      <th style={{verticalAlign: 'middle'}}>Update</th>
+      <th style={{verticalAlign: 'middle'}}>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+  
+  <tr>
+    <td rowSpan={2} style={{verticalAlign: 'middle'}}>
+      Vault Agent
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>ENHANCED</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Updated to use the latest Azure SDK version and Windows Identity
+      Federation (WIF).
+      <br /><br />
+      Learn more:&nbsp;
+      <a href="#">TBD</a>
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>GA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Fetch secrets directly into your application as environment variables. 
+      <br /><br />
+      Learn more: <a href="/vault/docs/agent-and-proxy/agent/process-supervisor">Process Supervisor Mode</a>
+    </td>
+  </tr>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      External plugins
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>BETA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Run external plugins in their own container with native container platform
+      controls.
+      <br /><br />
+      Learn more: <a href="/vault/docs/plugins/containerized-plugins">Containerize Vault plugins</a>
+    </td>
+  </tr>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      Eventing
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>BETA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Subscribe to notifications for various events in Vault. Includes support
+      for filtering, permissions, and cluster configurations with K-V secrets.
+      <br /><br />
+      Learn more: <a href="/vault/docs/concepts/events">Events</a>
+    </td>
+  </tr>
+
+  <tr>
```

Comment:
The audit logging work isn't customer facing, so I recommend removing this section.

Reply:
Removed